src/bluefin_v2_client/client.py

```diff
@@ -123,7 +123,11 @@ def create_order_to_sign(self, params: OrderSignatureRequest) -> Order:
         Returns:
             Order: order raw info
         """
-        expiration = current_unix_timestamp()
+        expiration = None
+        if "expiration" not in params:
+            expiration = current_unix_timestamp()
+        else:
+            expiration = params["expiration"]
```

Comment:
We should also validate the value of expiration field to avoid posting already expired orders (in case if user provides a past timestamp value for expiration)

Reply:
we already have the validation in dapi for that. should we include here as well?
`{'msg': 'placing stop limit order',
 'resp': {'error': {'code': 3014,
                    'message': 'Expiration time must be in the future.'}}}`